docs/api_docs/classes/great_expectations-data_context-data_context-data_context-DataContext.md

```diff
@@ -6,38 +6,39 @@ title: class DataContext
 
 ## Synopsis
 
-A DataContext represents a Great Expectations project. It organizes storage and access for
-expectation suites, datasources, notification settings, and data fixtures.
+A DataContext represents a Great Expectations project. It is the primary entry point for a Great Expectations
+deployment, with configurations and methods for all supporting components.
 
-The DataContext is configured via a yml file stored in a directory called great_expectations; the configuration file
-as well as managed expectation suites should be stored in version control.
+The DataContext is configured via a yml file stored in a directory called great_expectations; this configuration
+file as well as managed Expectation Suites should be stored in version control. There are other ways to create a
+Data Context that may be better suited for your particular deployment e.g. ephemerally or backed by GE Cloud
+(coming soon). Please refer to our documentation for more details.
 
-Use the `create` classmethod to create a new empty config, or instantiate the DataContext
-by passing the path to an existing data context root directory.
+DataContexts use Data Sources and Execution Engines that you're already familiar with to interact with data. You
```

Comment:
```suggestion
You
```

I'd just drop the first sentence here, especially since I don't think we should assume that most users are already familiar with Data Sources and Execution Engines.